src/code/templates/clutch-game.js

```diff
@@ -143,15 +142,21 @@ export default class ClutchGame extends Component {
     const parentGenomeClass = classNames('parent');
 
     function parentGenomeView(sex) {
+      let parentChangeableGenes;
+      // Changeable genes are either of the form ["wings, legs"] or [{mother: "wings", father:""}, {mother: "", father: "wings"]
```

Comment:
I took a go at your suggestion changes, @sfentress! I got rid of the confusing `changeable` and `staticDrake` arrays, and replaced it with an expanded `userChangeableGenes` object. I decided to let you define the `userChangeableGenes` in either the new way or the old way (just a flat array). The resulting code that handles either form is a little messy, what do you think of these changes?